### ISSUE TITLE
[QUESTION] Why are dates / datetimes surrounded with square brackets instead of angle brackets? 

### ISSUE THREAD

**Issue by LoveSponge** [vscode-org-mode/vscode-org-mode#131]
Hi

First of all thankyou all for this ext. It's awesome, works amazingly, and from what I can see, others here love it as well. Nice work!

Just a question as to why you decided to surround your dates and date times with square brackets instead of angle brackets?

```
[2017-01-01 Mon]
```

Instead of

```
<2017-01-01 Mon>
```
 ^ Which is how it is in real orgmode

And if it was just 'because' whether it would be okay if I amend this along with some other additilns? (mainly to do with subtree re-ordering)

Of course to combat existing users not being able to change their context I will update detection to accept both angle and square brackets as well. 

Thanks for reading in advance.

P.s if you're wondering why this matters at all, I'm using this extension to work with an application I use on my phone in order to cross-save org's between devices. The phone app (orgzly) uses angle brackets for dates and doesn't allow me to change it... 

_- LS_

**Comment by scaba** [comment]
This should explain it pretty well: https://orgmode.org/manual/Timestamps.html

TL;DR: square brackets tell org-agenda to ignore those dates and not show them in the agenda view

**Comment by LoveSponge** [comment]
Thanks @scaba, my own stupid fault for not reading the documentation thoroughly enough.

**Comment by pschwartz** [comment]
The docs noted show the reason behind the difference between <> and [], but is there a reason to not support both? It is very common in orgmode files to use <> for active dates for scheduling and open time tracking. 

I love the vscode extension, but it is a pain to not have it recognize my scheduling dates and then not be able to move the date with keybindings.

**Comment by LoveSponge** [comment]
@pschwartz i've forked this repo and will be submitting a pull request that adds support for context switching checkboxes (- [ ]) and scrolling dates with both '[]' and '<>'

I've currently got this working but have a little bit of tweaking to do.
(Currently if you increase the context date that is using '<>' brackets, it changes them to '[]' brackets. Currently, waiting to have the time to implement this.)

**Comment by sipi** [comment]
related PR #132
